src/xtgeo/common/calc.py

```diff
@@ -204,10 +204,13 @@ def point_in_tetrahedron(x0, y0, z0, vertices):
 
 
 def point_in_hexahedron(x0, y0, z0, vertices, _algorithm=1):
-    """Check if point P0 is inside a tetrahedron.
+    """
+    Check if point P0 is inside a tetrahedron.
```

Comment:
When using pydocstyle linting with google as style, I think(?) the style
`"""Use this form.
`
is mandatory, not

```
"""
Use this form.

```